sys/include/color.h

```diff
@@ -104,6 +105,30 @@ void color_str2rgb(const char *str, color_rgb_t *color);
  */
 void color_rgb2str(const color_rgb_t *rgb, char *str);
 
+/**
+ * @brief Invert a given rgb color
+ *
+ * @param[in] rgb       Input rgb color, that should be converted.
+ * @param[out] inv_rgb  Output rgb color, result of the conversion.
+ */
+static inline void color_rgb_invert(const color_rgb_t *rgb, color_rgb_t *inv_rgb)
+{
+    inv_rgb->r = rgb->r ^ 0xFF;
+    inv_rgb->g = rgb->g ^ 0xFF;
+    inv_rgb->b = rgb->b ^ 0xFF;
+}
+
+/**
+ * @brief Calculate the complementary color of a given rgb color.
+ *
+ * @note                Complementary color calculation according to adobe illustator calculations.
+ *                      See https://helpx.adobe.com/illustrator/using/adjusting-colors.html
+ *
+ * @param[in] rgb       Input rgb color.
+ * @param[out] comp_rgb Output rgb color, result of the complementary color calculation.
+ */
+void color_rgb_complementary(const color_rgb_t *rgb, color_rgb_t *comp_rgb);
```

Comment:
Now that you have added unittests: Documenting the precondition that both `rgb` and the out variable can't be `NULL` would be great for both of these functions.